subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheIntegrationTest.groovy

```diff
@@ -25,22 +25,29 @@ import spock.lang.Issue
 
 class ConfigurationCacheIntegrationTest extends AbstractConfigurationCacheIntegrationTest {
 
-    def "configuration cache for help on empty project"() {
+    def "configuration cache for '#task' on empty project"() {
         given:
         settingsFile.createFile()
-        configurationCacheRun "help"
+        configurationCacheRun(task, *options)
         def firstRunOutput = removeVfsLogOutput(result.normalizedOutput)
-            .replaceAll(/Calculating task graph as no configuration cache is available for tasks: help\n/, '')
+            .replaceAll(/Calculating task graph as no configuration cache is available for tasks: ${task}.*\n/, '')
             .replaceAll(/Configuration cache entry stored.\n/, '')
 
         when:
-        configurationCacheRun "help"
+        configurationCacheRun(task, *options)
         def secondRunOutput = removeVfsLogOutput(result.normalizedOutput)
             .replaceAll(/Reusing configuration cache.\n/, '')
             .replaceAll(/Configuration cache entry reused.\n/, '')
 
         then:
         firstRunOutput == secondRunOutput
+
+        where:
+        task            | options
+        "help"          | []
+        "properties"    | []
+        "dependencies"  | []
+        "help"          | ["--task", "help"]
     }
```

Comment:
@bamboo I remember you had some feedback about this test. My goal here was to have a cheap check for many help-plugin related tasks for an empty project.